pubpub/pubpub
# update & fix: DOI Deposit Language & Date

Makes a few more quick language updates, as pointed out by @jeffpooley. Principally:

- Capitalized Pub in language.
- Changes the suffix update/generate buttons to specify the prefix.
- Changes the helper text on successful suffix updates to indicate that a deposit is still needed to take effect.

This also fixes a bug Jeff found with Safari, which was throwing an invalid date error and crashing the client app. This appears to be because Safari doesn't support generating dates from arrays, ie `[MM, DD, YYYY]`. To fix this, I've changed the dateFormat input to a `YYYY-MM-DD` date string, and added the flag to handle it as UTC. I think this is right, because the release date comes from the database, which is UTC, and it's an absolute date being sent to Crossref, also UTC. But datemaster @idreyn should take a quick peek.

_Test plan_
- Visit a released Pub in Safari
- Generate a DOI preview, verify that publication date in deposit preview matches first release date.
- Note this date.
- Submit a DOI and make sure it's successful.
- Repeat in IE, Firefox, Chrome.
- Make sure publication date in deposit preview
